docs/fundamentals/code-analysis/quality-rules/ca1801.md

```diff
@@ -45,6 +45,10 @@ This rule does not examine the following kinds of methods:
 
 This rule does not flag parameters that are named with the [discard](../../../csharp/discards.md) symbol, for example, `_`, `_1`, and `_2`. This reduces warning noise on parameters that are needed for signature requirements, for example, a method used as a delegate, a parameter with special attributes, or a parameter whose value is implicitly accessed at run time by a framework but is not referenced in code.
 
+> [!NOTE]
+> This rule has been deprecated in favor of [IDE0060](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0060). You can enforce this analyzer by following [this guide](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/overview#code-style-analysis).
+> For more information, see [Deprecated rules](https://docs.microsoft.com/visualstudio/code-quality/fxcop-unported-deprecated-rules).
```

Comment:
- Do we need a parallel PR to MicrosoftDocs/visualstudio-docs page?
- The linked page says deprecated rules for FxCop, which was deleted recently, so the change only affects NetAnalyzers package. Do the linked page needs to be updated regarding this?

Reply:
I'd just suggest leaving this part out for now (for more info, see deprecated rules).

Reply:
Also, those deprecated rules refer to FxCop rules that were never ported as an analyzer. CA1801 was ported, so I don't think it belongs in that list.